docs/Push-To-Try.md

```diff
@@ -0,0 +1,53 @@
+Prerequisites
+============
+
+* gecko-dev git repo is checkout out and updated
+* mozilla-central mercurial repo is checked out and setup
+* moz-git-tools git repo is checked out and installed on your system
+* Make sure .hg/hgrc file in mozilla-central on your local has these additions added under the "default=":
+    * Add (without quotes) "default-push = ssh://hg.mozilla.org/mozilla-central”
+    * Add (without quotes) "try = ssh://hg.mozilla.org/try”
+* At least Level 1 commit access, if you have not requested this:
+  https://www.mozilla.org/en-US/about/governance/policies/commit/
+
+
+Push To Try
+============
+
+Commit your local changes
+On Local Loop Branch:
+```shell
+$ git commit -m “Bug 123456789 - whatever message you want” .
+```
+
+Update fx-team on gecko-dev to latest, remove old branch if exists and create new branch for your changes
+On gecko-dev/fx-team:
```

Comment:
Markdown is putting these two lines on the same line - I think you need an additional blank line between them (in several places).